elasticgraph-schema_definition/spec/unit/elastic_graph/schema_definition/runtime_metadata/index_definitions_by_name_spec.rb

```diff
@@ -258,7 +264,9 @@ module SchemaDefinition
             s.object_type "Component" do |t|
               t.field "id", "ID!"
               t.field "widget_fields", "ComponentWidgetFields"
-              t.index "components"
+              t.index "components" do |i|
+                i.has_had_multiple_sources!
+              end
```

Comment:
```suggestion
              t.index "components"
```

I don't believe it's needed here.

Reply:
thanks for catching these, updated.